lib/pure/os.nim

```diff
@@ -2795,13 +2795,14 @@ when defined(nimdoc):
   proc paramStr*(i: int): string {.tags: [ReadIOEffect].} =
     ## Returns the `i`-th `command line argument`:idx: given to the application.
     ##
-    ## `i` should be in the range `1..paramCount()`, the `IndexDefect`
-    ## exception will be raised for invalid values.  Instead of iterating over
-    ## `paramCount() <#paramCount>`_ with this proc you can call the
-    ## convenience `commandLineParams() <#commandLineParams>`_.
+    ## `i` should be in the range `start..paramCount()`, the `IndexDefect`
+    ## exception will be raised for invalid values. start is 1 for c/cpp backend, 
+    ## 2 for nodejs backend. Instead of iterating over `paramCount() <#paramCount>`_
```

Comment:
Start should always be 1...

Reply:
for nodejs, the first two elements are node and the path to your script.

E:\node\node.exe
D:\QQPCmgr\Desktop\Nim\test.js